docs/spaces.md

```diff
@@ -1,4 +1,4 @@
- Add or remove spaces here. To be used for trivial PRs from first-time contributors.
```

Comment:
I meant that this file can be changed in a separate PR first. This will let you run validation checks faster on subsequent PRs.

Reply:
Do you mean that for each new pull request, I can change this file first. And then add other changes?